### ISSUE TITLE
Minor mistake in Rekor docs

### ISSUE THREAD

**Issue by amouat** [sigstore/sigstore-website#100]
The website says:

```
Grab the rekor source:

go get -u -t -v github.com/sigstore/rekor/cmd/server

Note: You can also git clone should you prefer.
```

This doesn't work, I think this should be `github.com/sigstore/rekor/cmd/rekor-server`. It would also be nice to include the full `git clone` command.

Perhaps more importantly, the "Edit this page on github" link, doesn't work, should I open an issue for that?

**Comment by dlorenc** [comment]
> Perhaps more importantly, the "Edit this page on github" link, doesn't work, should I open an issue for that?

Oof, thats not good. Yeah, a separate issue for that would be great.

**Comment by znewman01** [comment]
This can be closed (separate issue mentioned above is #101).
